inst/templates/widget_js_new.txt

```diff
@@ -0,0 +1,19 @@
+HTMLWidgets.widget({
+
+  name: '%s',
+
+  type: 'output',
+
+  factory: function(el, width, height) {
+
+    return {
```

Comment:
Good point. I will add spacing as well as comments to indicate what each of the functions do.